frontend/src/metabase/admin/datamodel/components/database/ColumnItem.jsx

```diff
@@ -227,7 +227,11 @@ export class SpecialTypeAndTargetPicker extends Component {
         // handle a "secondary" input more elegantly
         showCurrencyTypeSelect && (
           <Select
-            className={cx("TableEditor-field-target inline-block", className)}
+            className={cx(
+              "TableEditor-field-target inline-block",
+              selectSeparator ? "mt0" : "mt1",
```

Comment:
Using the same conditional here to add spacing above the secondary select only when it's stacked vertically below the other select on the table metadata page.